RestCommClient/Classes/RCDevice.h

```diff
@@ -125,22 +125,17 @@ extern NSString* const RCDeviceCapabilityClientNameKey;
  *    <b>turn-password</b>: TURN password <br>
  *    <b>registrar</b>: Restcomm instance to use, like <i>'cloud.restcomm.com'</i>. Leave empty for registrar-less mode <br>
  *    <b>signaling-secure</b>: Should signaling traffic be encrypted? For traffic to be encrypted the 'signaling-certificate-dir' below should be provided as well <br>
- *    <b>signaling-certificate-dir</b>: Directory where files agent.pem and cafile.pem are installed inside the App bundle needed when TLS is to be used for signaling. agent.pem contains the private key and certificate to be used by the signaling facilities. cafile.pem is a file containing all trusted certs
+ *    <b>signaling-certificate-dir</b>: Directory where files agent.pem and cafile.pem are installed inside the App bundle needed when TLS is to be used for signaling.
+      agent.pem contains the private key and certificate to be used by the signaling facilities. cafile.pem is a file containing all trusted certs
  *    <b>ice-config-type<b>: Should be integer value of the following  kXirsysV2 = 0, kXirsysV3 =1 , kCustom=2
-      for example, if we want to use XirsysV2 server value should be 1
-
+      for example, if we want to use XirsysV2 server value should be 0
+ *    <b>ice-domain</b> If ice-config-type is kXirsysV3, ice-domain must be provided; for example: restcomm
  *    <b>ice-servers</b> NSArray of NSDictionary data of the STUN and TURN servers; This key is mandatory if ice-config-type is 'kCustom'; Example with 2 servers:
-             NSDictionary *dictionaryServer = [[NSDictionary alloc] initWithObjectsAndKeys:
-             @"46560f8e-94a7-11e7-FAKE", @"username",
-             @"turn:iceSERVER:80?transport=udp", @"url",
-             @"4656101a-94a7-11e7-97ac-FAKE", @"credential",
-             nil];
-             
-             NSDictionary *dictionaryServer2 = [[NSDictionary alloc] initWithObjectsAndKeys:
-             @"stun:ice1.SERVER",@"url", nil];
- 
             parameter to pass (Array):
-            @[dictionaryServer, dictionaryServers2]
+            @[
```

Comment:
I think the literal for NSDictionary needs the 

@{ @"key" : @"value" }

notation @ognjenns. I think your notation works only for `dictionaryWithObjectsAndKeys`, not literal. Can you please amend this? Here's the example I used previously to get an idea:

```
@[
   @{ @"url" : @"stun:turn01.uswest.xirsys.com" },
   @{ @"url" : @"turn:turn01.uswest.xirsys.com:80?transport=udp", @"username" : @"USERNAME_HERE", @"credential" : @"KEY_HERE" }
]
```

This is very handy as people tend to copy paste from doc into code